packages/sdk-core/test/5_subgraph.test.ts

```diff
@@ -36,5 +37,16 @@ describe("Subgraph Tests", () => {
         it("Should be able to use the listener", async () => {
             await testListenerInitialization(query);
         });
+
+        it("Should have the correct subgraph endpoints", async () => {
```

Comment:
Just a sanity check to ensure that the endpoints returned from the resolverData map is correct